[2024-11-20T14:46:56.550+0000] {processor.py:186} INFO - Started process (PID=1960) to work on /opt/airflow/dags/etl/load.py
[2024-11-20T14:46:56.552+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/etl/load.py for tasks to queue
[2024-11-20T14:46:56.555+0000] {logging_mixin.py:190} INFO - [2024-11-20T14:46:56.555+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/etl/load.py
[2024-11-20T14:46:56.701+0000] {logging_mixin.py:190} INFO - [2024-11-20T14:46:56.690+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/etl/load.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/etl/load.py", line 6, in <module>
    from etl.load_female_daily import insert_female_daily_data_into_postgres
ModuleNotFoundError: No module named 'etl.load_female_daily'
[2024-11-20T14:46:56.707+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/etl/load.py
[2024-11-20T14:46:56.740+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/etl/load.py took 0.201 seconds
[2024-11-20T14:47:27.471+0000] {processor.py:186} INFO - Started process (PID=2025) to work on /opt/airflow/dags/etl/load.py
[2024-11-20T14:47:27.473+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/etl/load.py for tasks to queue
[2024-11-20T14:47:27.476+0000] {logging_mixin.py:190} INFO - [2024-11-20T14:47:27.475+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/etl/load.py
[2024-11-20T14:47:27.566+0000] {logging_mixin.py:190} INFO - [2024-11-20T14:47:27.555+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/etl/load.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/etl/load.py", line 6, in <module>
    from etl.load_female_daily import insert_female_daily_data_into_postgres
ModuleNotFoundError: No module named 'etl.load_female_daily'
[2024-11-20T14:47:27.572+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/etl/load.py
[2024-11-20T14:47:27.602+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/etl/load.py took 0.146 seconds
[2024-11-20T14:47:57.745+0000] {processor.py:186} INFO - Started process (PID=2091) to work on /opt/airflow/dags/etl/load.py
[2024-11-20T14:47:57.748+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/etl/load.py for tasks to queue
[2024-11-20T14:47:57.750+0000] {logging_mixin.py:190} INFO - [2024-11-20T14:47:57.749+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/etl/load.py
[2024-11-20T14:47:57.817+0000] {logging_mixin.py:190} INFO - [2024-11-20T14:47:57.809+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/etl/load.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/etl/load.py", line 6, in <module>
    from etl.load_female_daily import insert_female_daily_data_into_postgres
ModuleNotFoundError: No module named 'etl.load_female_daily'
[2024-11-20T14:47:57.819+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/etl/load.py
[2024-11-20T14:47:57.847+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/etl/load.py took 0.110 seconds
[2024-11-20T14:48:28.101+0000] {processor.py:186} INFO - Started process (PID=2156) to work on /opt/airflow/dags/etl/load.py
[2024-11-20T14:48:28.103+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/etl/load.py for tasks to queue
[2024-11-20T14:48:28.106+0000] {logging_mixin.py:190} INFO - [2024-11-20T14:48:28.105+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/etl/load.py
[2024-11-20T14:48:28.271+0000] {logging_mixin.py:190} INFO - [2024-11-20T14:48:28.254+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/etl/load.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/etl/load.py", line 6, in <module>
    from etl.load_female_daily import insert_female_daily_data_into_postgres
ModuleNotFoundError: No module named 'etl.load_female_daily'
[2024-11-20T14:48:28.278+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/etl/load.py
[2024-11-20T14:48:28.309+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/etl/load.py took 0.222 seconds
[2024-11-20T14:48:58.773+0000] {processor.py:186} INFO - Started process (PID=2221) to work on /opt/airflow/dags/etl/load.py
[2024-11-20T14:48:58.776+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/etl/load.py for tasks to queue
[2024-11-20T14:48:58.779+0000] {logging_mixin.py:190} INFO - [2024-11-20T14:48:58.778+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/etl/load.py
[2024-11-20T14:48:58.954+0000] {logging_mixin.py:190} INFO - [2024-11-20T14:48:58.933+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/etl/load.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/etl/load.py", line 6, in <module>
    from etl.load_female_daily import insert_female_daily_data_into_postgres
ModuleNotFoundError: No module named 'etl.load_female_daily'
[2024-11-20T14:48:58.961+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/etl/load.py
[2024-11-20T14:48:59.012+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/etl/load.py took 0.253 seconds
[2024-11-20T14:49:29.179+0000] {processor.py:186} INFO - Started process (PID=2285) to work on /opt/airflow/dags/etl/load.py
[2024-11-20T14:49:29.181+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/etl/load.py for tasks to queue
[2024-11-20T14:49:29.183+0000] {logging_mixin.py:190} INFO - [2024-11-20T14:49:29.182+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/etl/load.py
[2024-11-20T14:49:29.279+0000] {logging_mixin.py:190} INFO - [2024-11-20T14:49:29.268+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/etl/load.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/etl/load.py", line 6, in <module>
    from etl.load_female_daily import insert_female_daily_data_into_postgres
ModuleNotFoundError: No module named 'etl.load_female_daily'
[2024-11-20T14:49:29.282+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/etl/load.py
[2024-11-20T14:49:29.304+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/etl/load.py took 0.131 seconds
[2024-11-20T14:50:00.327+0000] {processor.py:186} INFO - Started process (PID=2350) to work on /opt/airflow/dags/etl/load.py
[2024-11-20T14:50:00.329+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/etl/load.py for tasks to queue
[2024-11-20T14:50:00.331+0000] {logging_mixin.py:190} INFO - [2024-11-20T14:50:00.331+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/etl/load.py
[2024-11-20T14:50:00.407+0000] {logging_mixin.py:190} INFO - [2024-11-20T14:50:00.399+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/etl/load.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/etl/load.py", line 6, in <module>
    from etl.load_female_daily import insert_female_daily_data_into_postgres
ModuleNotFoundError: No module named 'etl.load_female_daily'
[2024-11-20T14:50:00.411+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/etl/load.py
[2024-11-20T14:50:00.438+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/etl/load.py took 0.119 seconds
[2024-11-20T14:50:30.547+0000] {processor.py:186} INFO - Started process (PID=2414) to work on /opt/airflow/dags/etl/load.py
[2024-11-20T14:50:30.549+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/etl/load.py for tasks to queue
[2024-11-20T14:50:30.551+0000] {logging_mixin.py:190} INFO - [2024-11-20T14:50:30.551+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/etl/load.py
[2024-11-20T14:50:30.634+0000] {logging_mixin.py:190} INFO - [2024-11-20T14:50:30.624+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/etl/load.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/etl/load.py", line 6, in <module>
    from etl.load_female_daily import insert_female_daily_data_into_postgres
ModuleNotFoundError: No module named 'etl.load_female_daily'
[2024-11-20T14:50:30.638+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/etl/load.py
[2024-11-20T14:50:30.665+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/etl/load.py took 0.126 seconds
[2024-11-20T14:51:01.302+0000] {processor.py:186} INFO - Started process (PID=2479) to work on /opt/airflow/dags/etl/load.py
[2024-11-20T14:51:01.312+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/etl/load.py for tasks to queue
[2024-11-20T14:51:01.316+0000] {logging_mixin.py:190} INFO - [2024-11-20T14:51:01.315+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/etl/load.py
[2024-11-20T14:51:01.453+0000] {logging_mixin.py:190} INFO - [2024-11-20T14:51:01.434+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/etl/load.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/etl/load.py", line 6, in <module>
    from etl.load_female_daily import insert_female_daily_data_into_postgres
ModuleNotFoundError: No module named 'etl.load_female_daily'
[2024-11-20T14:51:01.459+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/etl/load.py
[2024-11-20T14:51:01.508+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/etl/load.py took 0.217 seconds
[2024-11-20T14:51:31.612+0000] {processor.py:186} INFO - Started process (PID=2545) to work on /opt/airflow/dags/etl/load.py
[2024-11-20T14:51:31.615+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/etl/load.py for tasks to queue
[2024-11-20T14:51:31.618+0000] {logging_mixin.py:190} INFO - [2024-11-20T14:51:31.617+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/etl/load.py
[2024-11-20T14:51:31.707+0000] {logging_mixin.py:190} INFO - [2024-11-20T14:51:31.697+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/etl/load.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/etl/load.py", line 6, in <module>
    from etl.load_female_daily import insert_female_daily_data_into_postgres
ModuleNotFoundError: No module named 'etl.load_female_daily'
[2024-11-20T14:51:31.711+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/etl/load.py
[2024-11-20T14:51:31.755+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/etl/load.py took 0.153 seconds
[2024-11-20T14:52:02.027+0000] {processor.py:186} INFO - Started process (PID=2610) to work on /opt/airflow/dags/etl/load.py
[2024-11-20T14:52:02.030+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/etl/load.py for tasks to queue
[2024-11-20T14:52:02.032+0000] {logging_mixin.py:190} INFO - [2024-11-20T14:52:02.032+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/etl/load.py
[2024-11-20T14:52:02.215+0000] {logging_mixin.py:190} INFO - [2024-11-20T14:52:02.194+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/etl/load.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/etl/load.py", line 6, in <module>
    from etl.load_female_daily import insert_female_daily_data_into_postgres
ModuleNotFoundError: No module named 'etl.load_female_daily'
[2024-11-20T14:52:02.223+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/etl/load.py
[2024-11-20T14:52:02.276+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/etl/load.py took 0.257 seconds
[2024-11-20T14:52:11.750+0000] {processor.py:186} INFO - Started process (PID=2614) to work on /opt/airflow/dags/etl/load.py
[2024-11-20T14:52:11.752+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/etl/load.py for tasks to queue
[2024-11-20T14:52:11.755+0000] {logging_mixin.py:190} INFO - [2024-11-20T14:52:11.755+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/etl/load.py
[2024-11-20T14:52:11.795+0000] {logging_mixin.py:190} INFO - [2024-11-20T14:52:11.794+0000] {dagbag.py:366} INFO - File /opt/airflow/dags/etl/load.py assumed to contain no DAGs. Skipping.
[2024-11-20T14:52:11.796+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/etl/load.py
[2024-11-20T14:52:11.830+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/etl/load.py took 0.088 seconds
[2024-11-20T14:52:42.050+0000] {processor.py:186} INFO - Started process (PID=2679) to work on /opt/airflow/dags/etl/load.py
[2024-11-20T14:52:42.053+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/etl/load.py for tasks to queue
[2024-11-20T14:52:42.056+0000] {logging_mixin.py:190} INFO - [2024-11-20T14:52:42.056+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/etl/load.py
[2024-11-20T14:52:42.105+0000] {logging_mixin.py:190} INFO - [2024-11-20T14:52:42.105+0000] {dagbag.py:366} INFO - File /opt/airflow/dags/etl/load.py assumed to contain no DAGs. Skipping.
[2024-11-20T14:52:42.106+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/etl/load.py
[2024-11-20T14:52:42.133+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/etl/load.py took 0.092 seconds
[2024-11-20T14:53:12.384+0000] {processor.py:186} INFO - Started process (PID=2744) to work on /opt/airflow/dags/etl/load.py
[2024-11-20T14:53:12.386+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/etl/load.py for tasks to queue
[2024-11-20T14:53:12.389+0000] {logging_mixin.py:190} INFO - [2024-11-20T14:53:12.389+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/etl/load.py
[2024-11-20T14:53:12.423+0000] {logging_mixin.py:190} INFO - [2024-11-20T14:53:12.423+0000] {dagbag.py:366} INFO - File /opt/airflow/dags/etl/load.py assumed to contain no DAGs. Skipping.
[2024-11-20T14:53:12.424+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/etl/load.py
[2024-11-20T14:53:12.447+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/etl/load.py took 0.072 seconds
[2024-11-20T14:53:42.969+0000] {processor.py:186} INFO - Started process (PID=2809) to work on /opt/airflow/dags/etl/load.py
[2024-11-20T14:53:42.971+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/etl/load.py for tasks to queue
[2024-11-20T14:53:42.973+0000] {logging_mixin.py:190} INFO - [2024-11-20T14:53:42.973+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/etl/load.py
[2024-11-20T14:53:43.005+0000] {logging_mixin.py:190} INFO - [2024-11-20T14:53:43.004+0000] {dagbag.py:366} INFO - File /opt/airflow/dags/etl/load.py assumed to contain no DAGs. Skipping.
[2024-11-20T14:53:43.007+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/etl/load.py
[2024-11-20T14:53:43.032+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/etl/load.py took 0.071 seconds
[2024-11-20T14:54:13.280+0000] {processor.py:186} INFO - Started process (PID=2874) to work on /opt/airflow/dags/etl/load.py
[2024-11-20T14:54:13.282+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/etl/load.py for tasks to queue
[2024-11-20T14:54:13.284+0000] {logging_mixin.py:190} INFO - [2024-11-20T14:54:13.284+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/etl/load.py
[2024-11-20T14:54:13.326+0000] {logging_mixin.py:190} INFO - [2024-11-20T14:54:13.326+0000] {dagbag.py:366} INFO - File /opt/airflow/dags/etl/load.py assumed to contain no DAGs. Skipping.
[2024-11-20T14:54:13.327+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/etl/load.py
[2024-11-20T14:54:13.342+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/etl/load.py took 0.067 seconds
[2024-11-20T14:54:43.486+0000] {processor.py:186} INFO - Started process (PID=2939) to work on /opt/airflow/dags/etl/load.py
[2024-11-20T14:54:43.488+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/etl/load.py for tasks to queue
[2024-11-20T14:54:43.489+0000] {logging_mixin.py:190} INFO - [2024-11-20T14:54:43.489+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/etl/load.py
[2024-11-20T14:54:43.535+0000] {logging_mixin.py:190} INFO - [2024-11-20T14:54:43.535+0000] {dagbag.py:366} INFO - File /opt/airflow/dags/etl/load.py assumed to contain no DAGs. Skipping.
[2024-11-20T14:54:43.537+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/etl/load.py
[2024-11-20T14:54:43.553+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/etl/load.py took 0.071 seconds
[2024-11-20T14:55:14.299+0000] {processor.py:186} INFO - Started process (PID=3003) to work on /opt/airflow/dags/etl/load.py
[2024-11-20T14:55:14.301+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/etl/load.py for tasks to queue
[2024-11-20T14:55:14.303+0000] {logging_mixin.py:190} INFO - [2024-11-20T14:55:14.303+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/etl/load.py
[2024-11-20T14:55:14.333+0000] {logging_mixin.py:190} INFO - [2024-11-20T14:55:14.333+0000] {dagbag.py:366} INFO - File /opt/airflow/dags/etl/load.py assumed to contain no DAGs. Skipping.
[2024-11-20T14:55:14.334+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/etl/load.py
[2024-11-20T14:55:14.346+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/etl/load.py took 0.052 seconds
[2024-11-20T14:55:44.725+0000] {processor.py:186} INFO - Started process (PID=3068) to work on /opt/airflow/dags/etl/load.py
[2024-11-20T14:55:44.726+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/etl/load.py for tasks to queue
[2024-11-20T14:55:44.729+0000] {logging_mixin.py:190} INFO - [2024-11-20T14:55:44.728+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/etl/load.py
[2024-11-20T14:55:44.745+0000] {logging_mixin.py:190} INFO - [2024-11-20T14:55:44.745+0000] {dagbag.py:366} INFO - File /opt/airflow/dags/etl/load.py assumed to contain no DAGs. Skipping.
[2024-11-20T14:55:44.747+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/etl/load.py
[2024-11-20T14:55:44.761+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/etl/load.py took 0.046 seconds
[2024-11-20T14:56:15.604+0000] {processor.py:186} INFO - Started process (PID=3133) to work on /opt/airflow/dags/etl/load.py
[2024-11-20T14:56:15.605+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/etl/load.py for tasks to queue
[2024-11-20T14:56:15.607+0000] {logging_mixin.py:190} INFO - [2024-11-20T14:56:15.607+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/etl/load.py
[2024-11-20T14:56:15.646+0000] {logging_mixin.py:190} INFO - [2024-11-20T14:56:15.646+0000] {dagbag.py:366} INFO - File /opt/airflow/dags/etl/load.py assumed to contain no DAGs. Skipping.
[2024-11-20T14:56:15.648+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/etl/load.py
[2024-11-20T14:56:15.668+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/etl/load.py took 0.069 seconds
[2024-11-20T14:56:46.189+0000] {processor.py:186} INFO - Started process (PID=3198) to work on /opt/airflow/dags/etl/load.py
[2024-11-20T14:56:46.190+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/etl/load.py for tasks to queue
[2024-11-20T14:56:46.192+0000] {logging_mixin.py:190} INFO - [2024-11-20T14:56:46.192+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/etl/load.py
[2024-11-20T14:56:46.227+0000] {logging_mixin.py:190} INFO - [2024-11-20T14:56:46.226+0000] {dagbag.py:366} INFO - File /opt/airflow/dags/etl/load.py assumed to contain no DAGs. Skipping.
[2024-11-20T14:56:46.228+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/etl/load.py
[2024-11-20T14:56:46.240+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/etl/load.py took 0.056 seconds
